.travis.yml

```diff
@@ -20,6 +20,7 @@ jobs:
       python: 3.5 # version from Ubuntu 16.04
       install:
         - pip install mypy==0.780 pylint==2.4.4
+        - python3 -m pip install jinja2==2.10.1 types-jinja2
```

Comment:
We're adding a new mechanism to manage the python module requirements in a single place, instead of spread around various CI scripts. See #5197. Please replace the commit "Jinja2 prerequisite set up" by rebasing on top of `development` once #5197 as been merged.